### DISCUSSION TITLE
Welcome to ha-bambulab Discussions!

### DISCUSSION THREAD

**Discussion by greghesp** [greghesp/ha-bambulab#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by topgun0dk** [comment]
I can't find this add-in on hacs. Do I need to install it in another way ?

**Reply by greghesp** [reply]
I'll add to the readme. You can add the repo URL as a custom repository in HACS, then it'll show

**Reply by topgun0dk** [reply]
That was the hint i needed, thanks :)  (And wow a fast answer)

**Comment by Alex-DeIt** [comment]
Hello every body, I am trying to connect to my X1C via FTP unsing WinSCP. Unfortunately I can see only an empty directory (see here below)?
![image](https://github.com/user-attachments/assets/66527cd7-6ea4-401b-a9ad-4d06f8d265e4)
Can anyone give me a hint on how to access to the printer's directory?
